src/driver/aurora-mysql/AuroraMysqlDriver.ts

```diff
@@ -633,6 +633,9 @@ export class AuroraMysqlDriver implements Driver {
         ) {
             // convert to number if that exists in possible enum options
             value = parseInt(value)
+        } else if (columnMetadata.type === Number) {
+            // convert to number if number
+            value = !isNaN(+value) ? parseInt(value) : value
```

Comment:
I'm not sure why this change is in this PR?

Reply:
This change was made in all of the rdb drivers because it was not casting the data types automatically based on my models type definition (specifically for numbers) which hindered my development for this feature. If you are specifically asking about this driver, I can remove it for this one but did update all rdb drivers.

Reply:
Why did was parseInt chosen? this removes all decimals.